scripts/apothecary/formulas/openssl/openssl.sh

```diff
@@ -652,6 +652,7 @@ function copy() {
 		git checkout $1/include/openssl/opensslconf_osx.h
     	git checkout $1/include/openssl/opensslconf_android.h
     	git checkout $1/include/openssl/opensslconf_win32.h
+    	git checkout $1/include/openssl/opensslconf_vs.h
```

Comment:
this is incorrect, what we are doing is copying the original opensslconf.h as openssl_vs.h or whatever the platform is, as that file is generated in the config phase and is different for every platform. i've fixed it already but just so you know if you have to modify the script again. i'll add some comments in the script too